baseplate/experiments/providers/simple_experiment.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+import hashlib
+import time
+
+from .base import Experiment
+from ..._compat import long, iteritems
+
+
+logger = logging.getLogger(__name__)
+
+
+class SimpleExperiment(Experiment):
+    """Base class for simple experiments. Note that this class shares a lot of
+    code from the r2.py class, which is slated for deprecation and removal.
+    """
+
+    def make_seed(self, seed_data):
+        id = seed_data.get('id')
+        name = seed_data.get('name')
+        version = seed_data.get('shuffle_version')
+
+        return "{}.{}.{}".format(id, name, version)
+
+    def __init__(self, id, name, owner, start_ts, stop_ts, config,
+                 enabled=True, **kwargs):
+
+        self.id = id
+        self.name = name
+        self.owner = owner
+        self.num_buckets = 1000
+
+        self.start_ts = start_ts
+        self.stop_ts = stop_ts
+        self.enabled = enabled
+
+        self.bucket_val = config.get("bucket_val", "user_id")
+        self.version = config.get("experiment_version")
+        self.shuffle_version = config.get("shuffle_version")
+
+        self.variants = config.get("variants", [])
+
+        seed_data = {"id": id, "name": name, "shuffle_version": self.shuffle_version}
+        self.seed = config.get("bucket_seed", self.make_seed(seed_data))
+
+        self._validate_variants(self.variants)
+
+    def _get_seed(self):
+        return self.seed
```

Comment:
This seems like a weird pattern, the thing you are returning is "public" but this is "private", I think the way to do this would be something like:

```
@property
def seed(self):
    return self._seed
```

Reply:
Fixed